src/geetest/geetest.js

```diff
@@ -0,0 +1,74 @@
+// Based on https://github.com/GeeTeam/gt3-server-node-express-bypass/blob/master/app.js
+
+// https://docs.geetest.com/captcha/apirefer/api/server
+// doing this: "If the storage space is not sufficient: Send request to check bypass status before starting the verfication process."
+
+const axios = require("axios")
+const GeetestLib = require("./sdk/geetest_lib")
+
+async function sendRequest(params) {
+  const request_url = "http://bypass.geetest.com/v1/bypass_status.php"
+  let bypass_res
+  try {
+    const res = await axios({
+      url: request_url,
+      method: "GET",
+      timeout: 5000,
+      params: params,
+    })
+    const resBody = res.status === 200 ? res.data : ""
+    console.log(resBody)
+    bypass_res = resBody["status"]
+  } catch (e) {
+    bypass_res = ""
+  }
+  return bypass_res
+}
+
+class GeeTest {
+  constructor(GEETEST_ID, GEETEST_KEY) {
+    this.GEETEST_ID = GEETEST_ID
+    this.GEETEST_KEY = GEETEST_KEY
+  }
+
+  // private
+  async _getBypassStatus() {
+    const bypass_status = await sendRequest({ gt: this.GEETEST_ID })
+    return bypass_status
+  }
+
+  async register() {
+    const gtLib = new GeetestLib(this.GEETEST_ID, this.GEETEST_KEY)
+    const digestmod = "md5"
+    const userId = "test"
+    const params = {
+      digestmod: digestmod,
+      user_id: userId,
+      client_type: "web",
```

Comment:
The client type should be `"h5"` not `"web"` based on [this comment](https://github.com/GeeTeam/gt3-server-node-express-sdk/blob/80787f928563c6a5c1d401839bd7dadbb983be24/app.js#L28).